_columns.scss

```diff
@@ -123,17 +123,32 @@ $guss-columns-fallback-columns: 3 !default;
                     $css3-columns-support: $browser-supports-columns) {
     @if $css3-columns-support {
         #{$base-class} {
-            -webkit-column-width: rem($column-min-width);
-            -webkit-column-gap: rem($column-gap);
-            -webkit-column-fill: balance;
+            @if function-exists(rem) {
+                -webkit-column-width: rem($column-min-width);
+                -webkit-column-gap: rem($column-gap);
+                -webkit-column-fill: balance;
 
-            -moz-column-width: rem($column-min-width);
-            -moz-column-gap: rem($column-gap);
-            -moz-column-fill: balance;
+                -moz-column-width: rem($column-min-width);
+                -moz-column-gap: rem($column-gap);
+                -moz-column-fill: balance;
 
-            column-width: rem($column-min-width);
-            column-gap: rem($column-gap);
-            column-fill: balance;
+                column-width: rem($column-min-width);
+                column-gap: rem($column-gap);
+                column-fill: balance;
+            }
+            @else {
+                -webkit-column-width: $column-min-width;
+                -webkit-column-gap: $column-gap;
+                -webkit-column-fill: balance;
```

Comment:
I thought about it but I decided not the brake the pattern as it won't be duplicated in the final CSS and it looks good :-)

But as you've picked it up too I will move it out.